app/views/timeline_entries/new.html.erb

```diff
@@ -1 +1,35 @@
-<p>Placeholder</p>
+<%
+  links = [
+    {
+      text: 'Coronavirus pages',
+      href: coronavirus_pages_path
+    },
+    {
+      text: "#{@coronavirus_page.name} timeline entries",
+      href: coronavirus_page_path(slug: @coronavirus_page.slug)
+    },
+    {
+      text: "Add timeline entry"
+    },
+  ]
+
+  metadata = {
+    "Status" => "Draft",
+    "Last saved" => format_full_date_and_time(DateTime.now),
+  }
+%>
+
+<% content_for :breadcrumbs, render('shared/steps/step_breadcrumb', links: links) %>
+<% content_for :title, formatted_title(@coronavirus_page) %>
+<% content_for :context, "Add timeline entry" %>
+<% if @timeline_entry.errors.any? %>
+  <%= render "shared/sub_sections/form_errors", resource: @timeline_entry %>
+<% end %>
+
+<div class="covid-edit-page govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_for @timeline_entry, announcement: @timeline_entry, url: coronavirus_page_timeline_entries_path do  %>
```

Comment:
I don't think this "announcement" argument does anything, is it something caught in a copy & paste from announcements?

Reply:
I did wonder if you need the url and timeline_entry referenced here and discovered a TIL in the Rails manual.

It looks like `form_for` is soft deprecated and the conventional Rails way would be to use [`form_with`](https://edgeguides.rubyonrails.org/form_helpers.html). So this should probably be `<%= form_with url: coronavirus_page_timeline_entries_path do %>`

Reply:
You caught me with the copy and paste error. Oops.

Reply:
Hmmm. The form stops submitting when I switch to a `form_with`.

Reply:
Ah! Figured it out. I need to set `local: true`

Reply:
Oh gosh `local: true` sounds a strange attribute 🤔